### ISSUE TITLE
Changes to column style padding to improve visual layout of content

### ISSUE THREAD

**Issue by andrewleader** [Microsoft/AdaptiveCards#2755]
<table>
<tr>
<th>Release</th>
<th>Renderer status</th>
<th>Tasks status</th>
</tr>
<tr>
<td>1.2
<td>🔄 .NET (#2758)<br/>🔄 Android (#2759)<br/>🔄 iOS (#2760)<br/>🔄 TS (#2761)<br/>🔄 UWP (#2762)</td>
<td>🔄 Shared (#2756)<br/>🔄 Designer (#2757)</td>
</tr>
</table>

On the email thread with Shalini, David, Becky, and the rest of the team, we decided that we need to update the style/padding behavior of columns when only one of the columns uses a style. Details below...

Starting with a blank ColumnSet with three columns, it initially looks like this, which is good...

![image](https://user-images.githubusercontent.com/13246069/56753012-77612800-673e-11e9-935c-5014426dbd38.png)

But if you then change the first column to Emphasis, it looks like this... the text is no longer aligned! That's probably not the expected result.

**Actual**

![image](https://user-images.githubusercontent.com/13246069/56753090-a8d9f380-673e-11e9-88d1-c17c582a3887.png)

Instead, it should look like this (the rest of the columns also receive padding so that all the text in the columns are inline)

**Expected**

![image](https://user-images.githubusercontent.com/13246069/56753392-4c2b0880-673f-11e9-8035-082c0283bc79.png)

**Payload**

```json
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "ColumnSet",
            "columns": [ 
                {
                    "type": "Column",
                    "style": "emphasis",
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Emphasis"
                        }
                    ],
                    "width": "stretch"
                },
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Unset column"
                        }
                    ],
                    "width": "stretch"
                },
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Unset column"
                        }
                    ],
                    "width": "stretch"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.0"
}
```

Basically, if one of the columns has padding (as a result of a different style/background image), all of the other non-padded columns should receive padding.

Note that this IS a change from existing shipped behavior, but we can't think of any good reason why it should continue to behave the old way.

# Auto-generated task status

- [ ] Shared
- [ ] Designer
- [ ] .NET
- [ ] Android
- [ ] iOS
- [ ] TS
- [ ] UWP

**Comment by RebeccaAnne** [comment]
We have decided not to make this change as it has negative impacts on some existing cards.
